package.json

```diff
@@ -6,8 +6,9 @@
   "author": "Voluntera team",
   "license": "MIT",
   "scripts": {
-    "start": "NODE_ENV=development nodemon src/app.js",
-    "test": "NODE_ENV=test yarn validate && jest --passWithNoTests --runInBand",
+    "start": "NODE_ENV=production nodemon src/app.js",
+    "build": "yarn install",
+    "test": "yarn validate && jest --passWithNoTests",
```

Comment:
Don't modify the scripts necessary for local development. Create new scripts like `start-app` for production deployment.